test/data/bad/excessive_precision.geojson

```diff
@@ -0,0 +1 @@
+{ "type": "Point", "coordinates": [100.0000000001, 5.0000000001] }
```

Comment:
@perrygeo the spec notes that 6 decimal places is ~10cm

https://circle-artifacts.com/gh/geojson/draft-geojson/79/artifacts/0/home/ubuntu/draft-geojson/draft.html#rfc.section.10.2

How about using 6 in the test above?